test/periphery/fork/merkle-campaign/MerkleLT.t.sol

```diff
@@ -171,7 +179,22 @@ abstract contract MerkleLT_Fork_Test is Fork_Test {
             vars.merkleProof = getProof(leaves.toBytes32(), vars.leafPos);
         }
 
-        vars.merkleLT.claim({
+        // Expect call to `claim` with `sablierFee` as msg.value on the merkleLL contract.
+        vm.expectCall(
```

Comment:
the common function

Reply:
https://github.com/sablier-labs/v2-core/pull/1038/commits/ce35add0e51a934348f5f917460d126f8645506a